extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/pages/asset/AssetBuilder.java

```diff
@@ -83,6 +83,17 @@ private UiAssetCreateRequest buildCreateRequest(Asset asset, UiAssetEditMetadata
         createRequest.setTitle(editRequest.getTitle());
         createRequest.setTransportMode(editRequest.getTransportMode());
         createRequest.setVersion(editRequest.getVersion());
+        createRequest.setSovereignLegalName(editRequest.getSovereignLegalName());
+        createRequest.setGeoLocation(editRequest.getGeoLocation());
+        createRequest.setNutsLocation(editRequest.getNutsLocation());
+        createRequest.setDataSampleUrls(editRequest.getDataSampleUrls());
+        createRequest.setReferenceFileUrls(editRequest.getReferenceFileUrls());
+        createRequest.setReferenceFilesDescription(editRequest.getReferenceFilesDescription());
+        createRequest.setConditionsForUse(editRequest.getConditionsForUse());
+        createRequest.setDataUpdateFrequency(editRequest.getDataUpdateFrequency());
+        createRequest.setTemporalCoverageFrom(editRequest.getTemporalCoverageFrom());
+        createRequest.setTemporalCoverageToInclusive(editRequest.getTemporalCoverageToInclusive());
```

Comment:
lets test this in the UiApiWrapper Test properly, please.

Also ensure testing the edit metadata endpoint